x-pack/plugins/enterprise_search/server/routes/enterprise_search/connectors.ts

```diff
@@ -495,13 +495,7 @@ export function registerConnectorRoutes({ router, log }: RouteDependencies) {
     elasticsearchErrorHandler(log, async (context, request, response) => {
       const { client } = (await context.core).elasticsearch;
       const { connectorId } = request.params;
-      const { advanced_snippet, filtering_rules } = request.body;
```

Comment:
I'm confused whats happening with these parameters now, we still have `advanced_snippet` & `filtering_rules` on the request body but we're ignoring them now?

Reply:
Ahh, I forgot to clean up unused params from the request handler and associated logic, addressed it now :)